fe/fe-core/src/main/java/com/starrocks/alter/RollupJobV2.java

```diff
@@ -481,6 +484,22 @@ protected void runRunningJob() throws AlterCancelException {
                 } // end for tablets
             } // end for partitions
 
+            // colocate mv
+            ColocateTableIndex colocateTableIndex = GlobalStateMgr.getCurrentColocateIndex();
+            Set<String> colocateMaterializedViewNames = tbl.getColocateMaterializedViewNames();
+            if (!colocateTableIndex.isColocateTable(tableId) && colocateMaterializedViewNames.contains(rollupIndexName)) {
+                String groupName = db.getFullName() + ":" + rollupIndexName; ;
+                colocateTableIndex.addTableToGroup(dbId, tbl, groupName, null);
+                tbl.setInColocateMvGroup(true);
+                tbl.setColocateGroup(groupName);
```

Comment:
what if we have multiple mvs?

Reply:
With colocateMaterializedViewNames to record multiple mvs, make sure the base table is in the group as long as there is a colcoate mv